frontend/src/components/FAB/index.styles.ts

```diff
@@ -16,12 +16,12 @@ export const Container = styled.div`
   font-size: 30px;
   user-select: none;
   cursor: pointer;
-  position: absolute;
-  right: 0px;
+  position: fixed;
+  bottom: 50px;
```

Comment:
z-index가 없어서 아래 내용이 비치고 있네용
10 정도면 어떨까요?! 

```suggestion
  bottom: 50px;
  z-index: 10;
```

<img width="500" alt="스크린샷 2022-10-18 오후 3 09 32" src="https://user-images.githubusercontent.com/52737532/196349240-cafef533-9ca9-4535-88e8-4db9c96ca58d.png">